components/coprocessor_plugin_api/src/util.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
+use super::allocator::HostAllocatorPtr;
+use super::plugin_api::CoprocessorPlugin;
+
+/// Name of the exported constructor function for the plugin in the `dylib`.
+pub const PLUGIN_CONSTRUCTOR_NAME: &[u8] = b"_plugin_create";
+/// Type signature of the exported constructor function for the plugin in the `dylib`.
+pub type PluginConstructorSignature =
+    unsafe fn(host_allocator: HostAllocatorPtr) -> *mut dyn CoprocessorPlugin;
```

Comment:
If the return type is `Box<dyn CoprocessorPlugin>`, tikv will have a chance to call the `drop()` on plugin unload.

Reply:
Is there anything we should do about this?

Reply:
We can add it later when implementing unloading plugin.